cmd/kubectl-gadget/version.go

```diff
@@ -51,7 +51,7 @@ var versionCmd = &cobra.Command{
 		opts := metav1.ListOptions{LabelSelector: "k8s-app=gadget"}
 		pods, err := client.CoreV1().Pods("gadget").List(context.TODO(), opts)
 		if err != nil {
-			return commonutils.WrapInErrListNodes(err)
+			return commonutils.WrapInErrListPods(err)
```

Comment:
That's not the only place where it is wrongly used:

```bash
$ grep "CoreV1().Pods" -C 2 cmd/ -rn
cmd/kubectl-gadget/utils/trace.go-230-  opts := metav1.ListOptions{LabelSelector: "k8s-app=gadget"}
cmd/kubectl-gadget/utils/trace.go:231:  pods, err := client.CoreV1().Pods("gadget").List(context.TODO(), opts)
cmd/kubectl-gadget/utils/trace.go-232-  if err != nil {
cmd/kubectl-gadget/utils/trace.go-233-          return commonutils.WrapInErrListNodes(err)
[...]
cmd/kubectl-gadget/utils/exec.go:53:    pods, err := client.CoreV1().Pods("gadget").List(context.TODO(), listOptions)
cmd/kubectl-gadget/utils/exec.go-54-    if err != nil {
cmd/kubectl-gadget/utils/exec.go-55-            return err
[...]
cmd/kubectl-gadget/utils/flags.go:121:                  pods, err := client.CoreV1().Pods(GadgetNamespace).List(context.TODO(), opts)
cmd/kubectl-gadget/utils/flags.go-122-                  if err != nil {
cmd/kubectl-gadget/utils/flags.go-123-                          return commonutils.WrapInErrListNodes(err)
[...]
```

Reply:
Tried to convert all appropiate places. There are some in `pkg/container-collection` but there i can not import `commonutils`

Reply:
No, that error should be used only in the CLI.